src/packages/recompose-relay/package.json

```diff
@@ -18,8 +18,8 @@
     "lodash": "^4.0.0"
   },
   "peerDependencies": {
-    "recompose": "^0.17.0 || ^0.18.0 || ^0.19.0 || ^0.20.0",
+    "recompose": "^0.17.0 || ^0.18.0 || ^0.19.0 || ^0.20.0 || ^0.21.0 || ^0.22.0",
     "react": "^0.14.0 || ^15.0.0",
-    "react-relay": "^0.6.0 || ^0.7.0 || ^0.8.0 || ^0.9.0"
+    "react-relay": "^1.0.0-alpha1"
```

Comment:
I don't think so, granted that we change the import here, this new version of `recompose-relay` wouldn't be compatible with the previous versions of relay (there was no `react-relay/classic` export)